teachertool/src/style.overrides/Button.scss

```diff
@@ -75,3 +75,14 @@
 .pass > .common-button.common-dropdown-button {
     border: 3px solid var(--pxt-success-accent);
 }
+
+.undo-button {
+    width: 3rem;
+    height: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--pxt-info-accent-darkened);

```

Comment:
We should probably change these colors based on the type of toast containing them. I believe there are different css classes for each type of toast we can reference.

Reply:
Yeah, I think that's a good idea. Can this be done in a follow-up?

Reply:
Sure